src/adapter/src/catalog.rs

```diff
@@ -6899,9 +6899,11 @@ impl Catalog {
         PersistParameters {
             blob_target_size: Some(config.persist_blob_target_size()),
             blob_cache_mem_limit_bytes: Some(config.persist_blob_cache_mem_limit_bytes()),
+            compaction_enabled: Some(config.persist_compaction_enabled()),
             compaction_minimum_timeout: Some(config.persist_compaction_minimum_timeout()),
             consensus_connect_timeout: Some(config.crdb_connect_timeout()),
             consensus_tcp_user_timeout: Some(config.crdb_tcp_user_timeout()),
+            gc_enabled: Some(config.persist_gc_enabled()),
```

Comment:
Is my understanding correct that `gc_enabled` introduces a new feature flag and `compaction_enabled` replaces previous `MZ_PERSIST_COMPACTION_DISABLED`?

Reply:
if we merge this PR as-is, yes that's exactly right 👍